docker-compose.yml

```diff
@@ -1,20 +1,13 @@
 version: "3"
 services:
-  ambnode:
-    build:
-      context: .
-    container_name: "ambnode"
-    environment:
-      - NODE_ENV=production
-      - MONGODB_URI=mongodb://db:27017/ambrosus_node
-      - MONGODB_DATABASE=ambrosus_node
-      - WEB3_RPC=http://167.99.138.196:8545
-    ports:
-      - "80:9876"
-    depends_on:
-      - db
```

Comment:
Do I understand right that this docker-compose is only meant for development? If this is the case, I think it would be nice to have a comment/disclaimer on top.

Reply:
Yes, correct. For deployments I'm either going to use a docker file in a separate (private) repository or use Ansible's wrapper around Docker containers (http://docs.ansible.com/ansible/latest/modules/docker_module.html).

I will update with a comment.